ea-defs/config.go

```diff
@@ -157,3 +158,32 @@ func RequiredEADefsFor(cloud_type string) (res []ibclient.EADefinition) {
 	}
 	return
 }
+
+//Checks for cloud license in nios
+func CheckForCloudLicense(objMgr *ibclient.ObjectManager) {
+	flag, err := CheckLicense(objMgr, "cloud")
+	if err != nil {
+		fmt.Println("error", err)
+		os.Exit(4)
+	}
+	if !flag {
+		fmt.Println("Cloud License does not exist")
```

Comment:
Cloud License not available in Infoblox Appliance. Update and try again..

Reply:
done